azure-pipelines.yml

```diff
@@ -59,6 +59,16 @@ stages:
         python3 setup.py test
       displayName: "Unit tests"
 
+    - script: |
+        set -ex
+        # Install .NET CORE
+        curl -sSL https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+        sudo apt-add-repository https://packages.microsoft.com/debian/10/prod
+        sudo apt-get update
+        sudo apt-get -y install -f
```

Comment:
>sudo apt-get -y install -f [](http://example.com/codeflow?start=8&length=26)

Is this line really needed?

Reply:
From test result,  'sudo apt-get -y install -f' can and only can be set at this place.

Reply:
I tried these commands and removed `sudo apt-get -y install -f`. Seems working. Could you check again?

Reply:
yea，but failed again：
https://dev.azure.com/mssonic/build/_build/results?buildId=21794&view=logs&j=83516c17-6666-5250-abde-63983ce72a49&t=c10d5f44-55ce-55d7-7975-407ed75d9a96

Reply:
I compared the log with previous one, and the previous one seems suspicious
```
2021-06-29T03:04:13.8494132Z The following additional packages will be installed:
2021-06-29T03:04:13.8499448Z   libhiredis0.14
2021-06-29T03:04:13.8986264Z The following packages will be REMOVED:
2021-06-29T03:04:13.8993644Z   libibverbs-dev libnl-3-dev libnl-route-3-dev libopenmpi-dev
2021-06-29T03:04:13.9005991Z The following packages will be upgraded:
2021-06-29T03:04:13.9011224Z   libhiredis0.14
2021-06-29T03:04:13.9242794Z 1 upgraded, 0 newly installed, 4 to remove and 2 not upgraded.
```
The new apt source messed up the local package database. So it replaced our local built libhiredis0.14. I have not experiment, but propose several options:
1. Do not add apt source. Download the required deb packages and `sudo dpkg -i` them.
2. Or, add the apt source at the very beginning, before installing any other deb packages.

Reply:
update result, seems not work:
put add source at begin: 
https://dev.azure.com/mssonic/build/_build/results?buildId=21862&view=logs&j=83516c17-6666-5250-abde-63983ce72a49&t=c10d5f44-55ce-55d7-7975-407ed75d9a96
remove source and only wget debs: 
https://dev.azure.com/mssonic/build/_build/results?buildId=21866&view=logs&j=83516c17-6666-5250-abde-63983ce72a49&t=c10d5f44-55ce-55d7-7975-407ed75d9a96